nxt_editor/integration/__init__.py

```diff
@@ -3,24 +3,12 @@
 import subprocess
 import importlib
 
-import bpy
```

Comment:
Changes in this file are to undo foolish mistakes in https://github.com/nxt-dev/nxt_editor/pull/233